lib/include/lib/settings/general.hpp

```diff
@@ -146,6 +146,11 @@ namespace lib
 			 * \brief Ignore index of unavailable tracks
 			 */
 			bool ignore_unavailable_index = false;
+
+			/**
+			 * Swap song name and artist name order in queue (Artist - Song instead of Song - Artist)
+			 */
+			bool queue_swap_name_format = false;
```

Comment:
Adding new settings would be a breaking change that I'm not ready to merge right now, but I'm also not really sure if I like this, as it's a bit too specific